octoprint_slic3r/profile.py

```diff
@@ -38,148 +38,170 @@ class SeamPositions(object):
-
-    perimeter_acceleration=0,
-    infill_acceleration=0,
-    bridge_acceleration=0,
-    first_layer_acceleration=0,
-    default_acceleration=0,
-
-    layer_height=0.3,
-    first_layer_height=0.35,
-    infill_every_layers=1,
-    solid_infill_every_layers=0,
-
-    perimeters=3,
-    top_solid_layers=3,
-    bottom_solid_layers=3,
-    solid_layers=3,
-    fill_density="40%",
-    fill_angle=45,
-    fill_pattern=FillPatterns.HONEYCOMB,
-    solid_fill_pattern=FillPatterns.RECTILINEAR,
-    start_gcode=None,
-    end_gcode=None,
-    layer_gcode=None,
-    toolchange_gcode=None,
-    seam_position=SeamPositions.ALIGNED,
-    external_perimeters_first=False,
-    spiral_vase=False,
-    only_retract_when_crossing_perimeters=False,
-    solid_infill_below_area=70,
-    infill_only_where_needed=False,
-    infill_first=False,
-
-    extra_perimeters=True,
-    avoid_crossing_perimeters=False,
-    thin_walls=True,
-    overhangs=True,
-
-    support_material=False,
-    support_material_threshold=0,
-    support_material_pattern=SupportPatterns.HONEYCOMB,
-    support_material_spacing=2.5,
-    support_material_angle=0,
-    support_material_interface_layers=3,
-    support_material_interface_spacing=0,
-    raft_layers=0,
-    support_material_enforce_layers=0,
-    dont_support_bridges=True,
-
-    retract_length=1,
-    retract_speed=30,
-    retract_restart_extra=0,
-    retract_before_travel=2,
-    retract_lift=0,
-    retract_layer_change=True,
-    #wipe=False,
-
-    retract_length_toolchange=1,
-    retract_restart_extra_toolchange=1,
-
-    cooling=False,
-    min_fan_speed="35",
-    max_fan_speed="100",
-    bridge_fan_speed="100",
-    fan_below_layer_time=60,
-    slowdown_below_layer_time=30,
-    min_print_speed=10,
-    disable_fan_first_layers=1,
-    fan_always_on=False,
-
-    skirts=1,
-    skirt_distance=6,
-    skirt_height=1,
-    min_skirt_length=0,
-    brim_width=0,
-
-    scale=1, # TODO
-    rotate=0, # TODO
-    duplicate=1,
-    duplicate_grid=(1,1), # TODO ?!
-    duplicate_distance=6,
-    xy_size_compensation=0,
-
-    complete_objects=False,
-    extruder_clearance_radius=20,
-    extruder_clearance_height=20,
-
-    notes="",
-    resolution=0,
-
-    extrusion_width="100%",
-    first_layer_extrusion_width="100%",
-    perimeter_extrusion_width="100%",
-    external_perimeter_extrusion_width="100%",
-    infill_extrusion_width="100%",
-    solid_infill_extrusion_width="100%",
-    top_infill_extrusion_width="100%",
-    support_material_extrusion_width="100%",
-    bridge_flow_ratio=1,
-
-    extruder_offset="0x0",
-    perimeter_extruder=1,
-    infill_extruder=1,
-    support_material_extruder=1,
-    support_material_interface_extruder=1,
-    ooze_prevention=False,
-    standby_temperature_delta=-5
+    nozzle_diameter = 0.5,
+    print_center = (100, 100), # TODO
+    z_offset = 0,
+    gcode_flavor = GcodeFlavors.REPRAP,
+    use_relative_e_distances = 0, 			# 0 (unchecked) or 1 (checked)
```

Comment:
Does this hold true for *all* versions of slic3r? In any case the compatible and tested versions (plus OS) should be specified in the README. To much risk of version discrepancies otherwise I fear.